shared/oae/api/oae.api.js

```diff
@@ -132,6 +132,11 @@ define(['oae.api.admin', 'oae.api.authentication', 'oae.api.config', 'oae.api.co
                                         // initial widget loading have finished
                                         $('body').css('visibility', 'visible');
 
+                                        // Apply auto-focus after the core HTML widgets have loaded.
+                                        // The rendering will cause HTML5 autofocus to be lost, so
+                                        // we re-inforce it here
+                                        $('[autofocus]:first').focus();
```

Comment:
I always get the cursor at the beginning of the text field. The cursor should instead be positioned at the end of the value of the text field.